app/javascript/dashboard/helper/snoozeHelpers.js

```diff
@@ -1,11 +1,36 @@
-import { getUnixTime, format } from 'date-fns';
-import * as chrono from 'chrono-node';
+import { getUnixTime, format, add } from 'date-fns';
+import wootConstants from 'dashboard/constants/globals';
+
+const SNOOZE_OPTIONS = wootConstants.SNOOZE_OPTIONS;
 
 export const findSnoozeTime = snoozeType => {
-  const parsedDate = chrono.parseDate(snoozeType);
+  let parsedDate = null;
+
+  const currentDate = new Date();
+  const addTime = time => {
+    return new Date(
+      currentDate.getFullYear(),
+      currentDate.getMonth(),
+      currentDate.getDate() + time,
+      9,
+      0,
+      0,
+      0
+    );
+  };
+
+  if (snoozeType === SNOOZE_OPTIONS.AN_HOUR_FROM_NOW) {
+    parsedDate = add(currentDate, { hours: 1 });
+  } else if (snoozeType === SNOOZE_OPTIONS.UNTIL_TOMORROW) {
+    parsedDate = addTime(1);
+  } else if (snoozeType === SNOOZE_OPTIONS.UNTIL_NEXT_WEEK) {
+    parsedDate = addTime(7);
+  } else if (snoozeType === SNOOZE_OPTIONS.UNTIL_NEXT_MONTH) {
+    parsedDate = addTime(30);
```

Comment:
Another thought I had, what should "Next month" mean, should it be exactly one month from now, or the start of the next month?

GMail has options like these

<img width="245" alt="CleanShot 2023-06-14 at 15 10 04@2x" src="https://github.com/chatwoot/chatwoot/assets/18097732/e4bea978-4092-4fe2-88d0-bd3f0fe2e45a">

Here next week starts on Monday, also they have a Later this week option

Reply:
Yes. I agree. We can do something like,

Next week -> Next week's first working day, Monday at 9.00 AM.
Next month -> Next month, first working day, first Monday at 9.00 AM.

I have also considered adding expected time as an option, but it may require customization of the command bar component.